### ISSUE TITLE
Vault 1.6.9 Error and Reporting wrong Version

### ISSUE THREAD

**Issue by MCTyler** [Zrips/CMI#237]
Vault 1.6.9 Errors

#### IGNORE IF YOU ARE SUBMITTING A FEATURE REQUEST ####

ERROR:
```
####################
## [15:29:34] [Server thread/WARN]: [Vault] Task #21 for Vault v1.6.8 generated an exception
java.util.ConcurrentModificationException: null
	at java.util.HashMap$HashIterator.nextNode(HashMap.java:1442) ~[?:1.8.0_162]
	at java.util.HashMap$KeyIterator.next(HashMap.java:1466) ~[?:1.8.0_162]
	at org.bukkit.permissions.PermissibleBase.clearPermissions(PermissibleBase.java:168) ~[patched_1.12.2.jar:git-Paper-"344a125"]
	at org.bukkit.permissions.PermissibleBase.recalculatePermissions(PermissibleBase.java:149) ~[patched_1.12.2.jar:git-Paper-"344a125"]
	at org.bukkit.plugin.SimplePluginManager.dirtyPermissibles(SimplePluginManager.java:677) ~[patched_1.12.2.jar:git-Paper-"344a125"]
	at org.bukkit.plugin.SimplePluginManager.calculatePermissionDefault(SimplePluginManager.java:656) ~[patched_1.12.2.jar:git-Paper-"344a125"]
	at org.bukkit.plugin.SimplePluginManager.addPermission(SimplePluginManager.java:628) ~[patched_1.12.2.jar:git-Paper-"344a125"]
	at org.bukkit.plugin.SimplePluginManager.addPermission(SimplePluginManager.java:616) ~[patched_1.12.2.jar:git-Paper-"344a125"]
	at net.milkbowl.vault.Vault$1.run(Vault.java:157) ~[?:?]
	at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftTask.run(CraftTask.java:63) ~[patched_1.12.2.jar:git-Paper-"344a125"]
	at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:415) [patched_1.12.2.jar:git-Paper-"344a125"]
	at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:833) [patched_1.12.2.jar:git-Paper-"344a125"]
	at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:426) [patched_1.12.2.jar:git-Paper-"344a125"]
	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:767) [patched_1.12.2.jar:git-Paper-"344a125"]
	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:665) [patched_1.12.2.jar:git-Paper-"344a125"]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_162]##
####################
```
Even though it says Vault 1.6.8 up there it's 1.6.9

Cmi Version (using`/cmi version`): 7.6.5.2
Server Type (Spigot/Paperspigot/etc): PaperClip
Server Version (using `/ver`): git-Paper-"344a125" (MC: 1.12.2)
Vault: 1.6.9

**Comment by MCTyler** [comment]
Fixed by using Vault 1.6.6 from here https://www.spigotmc.org/resources/vault.34315/ and using the good ole' CMI Injector...Other may not have this option.

**Comment by Zrips** [comment]
Updated CMIVault, tho its uses same code as 1.6.6 so no clue why it would fail in first place. You can always use injector tho.
